packages/block-library/src/embed/index.js

```diff
@@ -138,21 +149,18 @@ export function getEmbedEdit( title, icon ) {
 		}
 
 		/**
-		 * Sets the appropriate CSS class to enforce an aspect ratio when the embed is resized
+		 * Gets the appropriate CSS class names to enforce an aspect ratio when the embed is resized
 		 * if the HTML has an iframe with width and height set.
 		 *
 		 * @param {string} html The preview HTML that possibly contains an iframe with width and height set.
+		 * @return {Object} Object with classnames set to true, for use with `classnames`.
```

Comment:
I don't quite understand what this means, I'm guessing it follows this shape:

`{ 'is-Mobile': isMobile, 'is-Responsive': isResponsive }`, etc.?

Maybe just:

```js
/* 
* @return {Object} Class names, for use with `classnames()`.
*/
```

would work?